isis/scripts/downloadIsisData

```diff
@@ -145,6 +145,11 @@ def create_rclone_arguments(destination, mission_name, parsedArgs, rclone_kwargs
         excludes = [f"- {arg}" for arg in args.exclude]
         filter_list.extend(excludes)    
 
+    if args.no_kernels:
+        excludes = [f"- {arg}" for arg in ['ck/**','ek/**','fk/**','ik/**','iak/**','lsk/**','mk/**','pck/**','sclk/**','spk/**']]
```

Comment:
add: `tspk/**`, `dsk/**`, 

some kernel groups have `orbnum/**` and `old/**` , and `SpecialProducts/**`, but we should add that to the global ignore list at the start. of the script.